### ISSUE TITLE
Tabbing order seems to have changed

### ISSUE THREAD

**Issue by catherinecarter** [phetsims/center-and-variability#576]
While investigating #564, I noticed the tabbing order seems to have changed for the interval tool.

Prior, after checking the interval tool checkbox while tabbing, the order was:
1. tab to the checkbox and toggle on
2. tab again, the left handle of the interval tool had focus
3. tab again, the right handle of the interval tool had focus
4. tab again, the interval tool block had focus

Now, the order has reversed (the original order seemed more logical):

https://github.com/phetsims/center-and-variability/assets/109630002/4eaaa6f8-6876-4bbb-bf2a-3d5f9b1c17c6

**Comment by marlitas** [comment]
It appears this is being caused by the option that was added for #564 `focusPanDirection`. I'm not exactly sure why that's causing the change, but I'll investigate.

I also wanted to mention that I believe the order has been handle1, block, handle2. That is what I see prior to the `focusPanDirection` change, and is also what I remember for a couple of weeks at least now...

**Comment by catherinecarter** [comment]
If possible, can we set the order to handle1, handle2, then the block? That seems to make more sense that you'd want to set the width using both handles before moving the block.

**Comment by catherinecarter** [comment]
The tabbing order looks great when first tabbing to the checkbox and then to the tool. 

Upon reset all and/or after moving one of the handles, the order seems to change. Here's a video and what I did:
1. Kick a few and tab to the checkbox
3. Tab to the tool. Correct order of handle1, handle 2, interval block, then on to the radio button group to the left of the accordion box
4. Repeat, all looks well (nothing has been moved)
5. Reset all
6. Tab to the checkbox
7. Tab to the tool and move the handle
8. Tab again, goes immediately to the radio button group rather than the other handle
9. Reset all
12. Tab to the checkbox
13. Tab to the tool - interval block is the first thing to have focus, followed by the radio button group

That's a lot of steps haha. Hopefully they make sense. Here's a video:

https://github.com/phetsims/center-and-variability/assets/109630002/d4a2032b-f4b9-4d85-a0b8-7823b1543cde

**Comment by marlitas** [comment]
This should be addressed @catherinecarter. Can you confirm on main and close if all looks well?

**Comment by catherinecarter** [comment]
Looks great in Chrome. 

I tested in Safari, and I could only tab into the play area and no where else. Here's a video where I tabbed once and focus went straight to the field. I tabbed, shift-tabbed, and tried to use space and the arrow keys, but focus remained in the field. Here's a video, although I'm sure it's straightforward to picture.

https://github.com/phetsims/center-and-variability/assets/109630002/c1b154fc-18f4-4766-b306-0606e63db0f8
